tests/conftest.py

```diff
@@ -74,14 +81,269 @@
 }
 
 
-TEST_DIALOGUES = [
-    "data/test_dialogues/default.json",
-    "data/test_dialogues/formbot.json",
-    "data/test_dialogues/moodbot.json",
-]
+TEST_DEFAULT_DIALOGUE = Dialogue(
```

Comment:
Optional: due to the amount of lines. How about moving it a module called `test_dialogues`? 

Optional2: I think we can skip this whole `Dialogue` stuff and just have list of `Event`s here. We can then create the trackers via `DialogueStateTracker.from_events`. I made this optional as it goes beyond the scope of the PR. In my opinion we can remove the entire `Dialogue` object as it has no benefit over using `DialogueStateTracker` / the events directly

Reply:
I'd go with your first option here, as it's a bit more work and I think it's a bit beyond the scope. I think `tests/dialogues.py` is enough, because naming it `test_dialogues` could be misleading (it's not a test file and shouldn't start with `test_`)

Reply:
sounds good 👍🏻